### ISSUE TITLE
[Request] Center User interface on several situations.

### ISSUE THREAD

**Issue by Vatsnik** [celluloid-player/celluloid#530]
Hi!

I just love Celluloid, but sadly it lacks some options that could improve it's experience, that's why (if possible), I would like to ask for the developers to implement it in the future.

Overview Description:

**Center user interface on initial load**
When enabled, Celluloid screen-centers the user interface (the player window) when loaded.

**Center user interface when loading new media**
When enabled, Celluloid screen-centers the user interface (the player window) when loading new media. (without crop), and adapts/extends to the media resolution

**Center user interface when exiting Fullscreen / Zoom mode**
When enabled, Celluloid screen-centers the user interface (the player window) when exiting fullscreen/zoom mode.

**Comment by gnome-mpv** [comment]
On Wayland, you can't position the window at all, so it isn't possible to implement any of these.

If you're using X11, you can already center the window on initial load using the mpv option `--geometry=50%:50%`. Right now changes to `--geometry` does not take effect until Celluloid is restarted, so the other cases are not yet possible.

**Comment by francoism90** [comment]
@gnome-mpv I'm running celluloid on Gnome Wayland and I'm unable to save the last window position. Is this the same bug? Because indeed using `--geometry` doesn't seem to work at all on Wayland.

Thanks!

**Comment by gnome-mpv** [comment]
>I'm running celluloid on Gnome Wayland and I'm unable to save the last window position. Is this the same bug? Because indeed using --geometry doesn't seem to work at all on Wayland.

Yes, it's the same bug. We can control the size of the window on Wayland, but not the position.

>Also could you please create/tag a new release? :)

Sure, I'll make a new release in a week or so.
